episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+![Early determination of the delay-adjusted confirmed case fatality ratio of severe acute respiratory syndrome (SARS) in Hong Kong, 2003.](fig/cfr-pone.0006852.g003-fig_d.png)
+
+We can explore this behaviour using the `cfr_rolling()` function. With `tail()`, we show that the latest CFR estimates are equal to the `cfr_static()` outputs:  
+
+```{r}
+# Calculate the rolling daily naive CFR
+rolling_cfr_naive <- cfr::cfr_rolling(data = ebola1976)
+
+tail(rolling_cfr_naive)
+```
+
+```{r}
+# Calculate the rolling daily delay-adjusted CFR
+rolling_cfr_adjusted <- cfr::cfr_rolling(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+
+tail(rolling_cfr_adjusted)
+```
+
+Now, let's visualise both results in a time series. How would the naive and delay-adjusted CFR estimates perform in real time?
+
+```{r,echo=FALSE,warning=FALSE,message=FALSE}
+
+# get the latest delay-adjusted CFR
+rolling_cfr_adjusted_end <-
+  rolling_cfr_adjusted %>%
+  dplyr::slice_tail()
+
+# bind by rows both output data frames
+bind_rows(
+  rolling_cfr_naive %>%
+    mutate(method = "naive"),
+  rolling_cfr_adjusted %>%
+    mutate(method = "adjusted")
+) %>%
+  # visualise both adjusted and unadjusted rolling estimates
+  ggplot() +
+  geom_ribbon(
+    aes(
+      date,
+      ymin = severity_low,
+      ymax = severity_high,
+      fill = method
+    ),
+    alpha = 0.2, show.legend = FALSE
+  ) +
+  geom_line(
+    aes(date, severity_mean, colour = method)
+  ) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_mean)) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_low), lty = 2) +
+  geom_hline(data = rolling_cfr_adjusted_end,
+             aes(yintercept = severity_high), lty = 2)
+```
+
+The horizontal line represents the delay-adjusted CFR estimated at the outbreak's end. The dotted line means the estimate has a 95% confidence interval (95% CI).
+
+:::::::::::::::::::::::::::::::::::::::::::: challenge
+
+### Interpret the early-stage CFR estimate
+
+Based on the figure above:
+
+- How much difference in days is between the date in which the 95% CI of the estimated _delay-adjusted CFR_ vs _naive CFR_ cross with the CFR estimated at the end of the outbreak?
+
+:::::::::::::::::::::: hint
+
+We can use either visual inspection or analysis of the output data frames.
+
+::::::::::::::::::::::
+
+:::::::::::::::::::::: solution
+
+There is almost one month of difference.
+
+Note that the estimate has considerable uncertainty at the beginning of the time series. After two weeks, the delay-adjusted CFR approaches the overall CFR estimate at the outbreak's end.
+
+Is this pattern similar to other outbreaks? We can use the data sets in this episode's challenges. We invite you to find it out!
+
+::::::::::::::::::::::
+
+
+::::::::::::::::::::::::::::::::::::::::::::
+
+
+
+::::::::::::::::: spoiler
+
+### rolling or time-varying?
```

Comment:
All uses of "sensible" should read "sensitive"; this is a false friend from Spanish/French <-> English.

Reply:
fixed in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/dd79a767861061cc5753691c40509c744f10adad